GoMain/src/main/main.go

```diff
@@ -74,10 +74,7 @@ func main() {
 	if err := orchestrationInit(); err != nil {
 		log.Fatalf("[%s] Orchestaration initalize fail : %s", logPrefix, err.Error())
 	}
-
-	for {
-		time.Sleep(1000)
-	}
+	sigmgr.Watch()
```

Comment:
@suresh-lc I think that this is quite valid because the previous implementation is hard-coded with the `Sleep` func with a specific value.